adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidADALTests.cs

```diff
@@ -103,7 +103,7 @@ public void AcquireTokenADFSv4FederatedInteractiveTest()
             user.FederationProvider = FederationProvider.AdfsV4;
             user.IsFederatedUser = true;
```

Comment:
You could convert this into a method on the helper, E.g. GetLabResponseWithADFSV4User ... or similar